mandatory/3-function-output.js

```diff
@@ -1,16 +1,21 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// Add comments to explain what this function does. You're meant to use Google! 
+/* Math.random returns a pseudo-random number between 0 - less than 1 (by default) 
+   and then scale it between whichever numbers you desire */
+
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+// This function is going to concatenate the string arguments to the calling string and then returns the new string
```

Comment:
You lost me. What is a calling string?

Reply:
The calling string is what comes before the `.concat` so in this case it would be `w1`

Reply:
Done, thanks for the feedback.